ui/app/ot/controller/calendarViewController.js

```diff
@@ -1,28 +1,33 @@
 'use strict';
 
 angular.module('bahmni.ot')
-    .controller('calendarViewController', ['$scope', '$rootScope', '$state', '$stateParams', 'appService', 'patientService', 'locationService', 'ngDialog',
-        function ($scope, $rootScope, $state, $stateParams, appService, patientService, locationService, ngDialog) {
+    .controller('calendarViewController', ['$scope', '$rootScope', '$state', '$stateParams', 'appService', 'patientService', 'locationService', 'ngDialog', 'surgicalAppointmentHelper',
+        function ($scope, $rootScope, $state, $stateParams, appService, patientService, locationService, ngDialog, surgicalAppointmentHelper) {
+            var CALENDAR_VIEW = 'Calendar';
             $scope.viewDate = $stateParams.viewDate || $state.viewDate || (moment().startOf('day')).toDate();
             $state.viewDate = $scope.viewDate;
             $scope.calendarConfig = appService.getAppDescriptor().getConfigValue("calendarView");
-
+            var weekStartDay = appService.getAppDescriptor().getConfigValue('startOfWeek') || Bahmni.OT.Constants.defaultWeekStartDayName;
+            var currentDate = moment().startOf('day').toDate();
```

Comment:
@buvaneswari-arun its basic moment() that we are using and nothing specific to dates the we are using here. I feel it doesn't have to go to DateUtils.js file.